### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds several unit tests**

Also:
- ```cl.events.js``` was refactored to improve testability
- ```calendar.html``` was updated for usage in unit testing

### CODE REVIEW

**Comment by FinalAngel** [comment]
@mkoistinen please do not merge before we have a "LGTM" or "+1"

**Comment by mkoistinen** [comment]
I think the rule is "don't merge until its been reviewed." I reviewed it, then merged it.

File: aldryn_events/boilerplates/bootstrap3/static/js/addons/cl.events.js [hunk 1]
```diff
@@ -23,56 +23,57 @@ var Cl = window.Cl || {};
             });
         },
 
-        // INFO: handle calendar
-        calendar: function (calendar) {
+        _handler: function (e) {
```
Comment (FinalAngel) [inline aldryn_events/boilerplates/bootstrap3/static/js/addons/cl.events.js]: I think you had to rewrite this to test the functionality better @Marketionist /cc @vxsx

File: aldryn_events/tests/frontend/unit/test.cl.events.js [hunk 2]
```diff
@@ -3,20 +3,50 @@
  * @copyright: http://www.divio.ch
  */
 
-'use strict';
+(function () {
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: why did you add this, @vxsx you suggested to remove this or?

File: aldryn_events/tests/frontend/unit/test.cl.events.js [hunk 3]
```diff
@@ -3,20 +3,50 @@
...
+        beforeEach(function () {
+            fixture.setBase('frontend/fixtures');
+            this.markup = fixture.load('calendar.html');
+        });
+
+        afterEach(function(){
+            fixture.cleanup();
+        });
+
+        it('Cl namespace is available', function () {
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: we probably need to change the naming here "cl.events.js Cl namespace is available" does not sound right

File: aldryn_events/tests/frontend/unit/test.cl.events.js [hunk 4]
```diff
@@ -3,20 +3,50 @@
...
+            Cl.events.init();
+
+            // validate that calendar was called inside Cl.events.init()
+            expect(Cl.events.calendar).toHaveBeenCalled();
+            expect(Cl.events.calendar.calls.count()).toEqual(1);
+        });
+
+        it('Cl.events._handler returns false if direction is not specified', function () {
+            expect(Cl.events._handler.call(
+                $('.js-trigger')[0],
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: it would be better to use `$('.js-trigger').get(0)` but I'm also not sure /cc @vxsx

File: aldryn_events/boilerplates/bootstrap3/static/js/addons/cl.events.js (hunk 1 above)
Comment (Marketionist) [inline aldryn_events/boilerplates/bootstrap3/static/js/addons/cl.events.js]: Yeah, had to rewrite to make it testable

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 2 above)
Comment (vxsx) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: in general I don't like this in tests, but jshint will complain and it makes sense to be consistent everywhere

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 4 above)
Comment (vxsx) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: we never really use `$.fn.get` sadly, in this case it's the same thing, `[0]` being a tiniest bit faster, I wouldn't change it

File: aldryn_events/tests/frontend/unit/test.cl.events.js [hunk 5]
```diff
@@ -3,20 +3,50 @@
...
+                { preventDefault: function () {} })
+            ).toEqual(false);
```
Comment (vxsx) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: i hope there are more tests coming, this is the simplest of the outcomes

Comment (Marketionist) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: Yes, sure, more test ahead ;)

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 3 above)
Comment (Marketionist) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: Ok, will be changed in the next PR

Comment (Marketionist) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: How about 'has available Cl namespace'?

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 2 above)
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: alright :+1:

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 3 above)
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: yep :+1:

File: aldryn_events/tests/frontend/unit/test.cl.events.js (hunk 4 above)
Comment (FinalAngel) [inline aldryn_events/tests/frontend/unit/test.cl.events.js]: alright thanks for clarifying
